docs/wiki_flu.md

```diff
@@ -0,0 +1,78 @@
+# New axis system for geometries, devices, and PROTOs
+
+Starting from the R2022a version, all geometries, devices, and PROTOs that come with Webots use the FLU axis orientation (x-**F**orward, y-**L**eft, and z-**U**p, see [#3073](https://github.com/cyberbotics/webots/pull/3073)).
+We introduced the update to be consistent with ROS (see [REP 103](https://www.ros.org/reps/rep-0103.html)) and other robotics systems.
+If your Webots world uses Webots geometries, devices, and PROTOs then you will probably need to adapt your world.
+
+[add image example of a 2021b world with the axis and a 2022a world with the axis]
```

Comment:
It would be nice to actually add this image now.

Reply:
![spot2](https://user-images.githubusercontent.com/48200998/145968280-5579900f-949d-49e5-8e69-d9c0f0f48b32.png)
Example of spot.wbt in 2021b: spot is RUB and the world is NUE.
![spot1](https://user-images.githubusercontent.com/48200998/145968288-5f697a88-68c0-4267-b569-582605702acb.png)
Example of spot.wbt in 2022a: spot is FLU and the world is ENU.